package.json

```diff
@@ -64,11 +64,9 @@
     "husky": "4.2.5",
     "jest": "26.0.1",
     "lint-staged": "10.2.10",
-    "playwright": ">=0.12.1",
-    "playwright-chromium": ">=0.12.1",
-    "playwright-core": ">=0.12.1",
-    "playwright-firefox": ">=0.12.1",
-    "playwright-webkit": ">=0.12.1",
+    "playwright": ">=1.1.1",
+    "playwright-chromium": ">=1.1.1",
+    "playwright-core": "npm:playwright-chromium@1.1.1",
```

Comment:
shouldn't it be `>=1.1.1`?

Reply:
I told about `"playwright-core": ">=1.1.1"` :)

Reply:
Thats the workaround for https://github.com/microsoft/playwright/issues/2540

Reply:
> Thats the workaround for microsoft/playwright#2540

got it. I think we should keep #144opened while it won't be fixed in **playwright**